### PR TITLE
chore: fix broken links

### PR DESCRIPTION
**Pull request by rafaelmagalhaes-ctw**
## What this PR changes/adds

The links in the main readme were updated as they were broken. To fix some broken links, the docs/developer/decision-records directory had to be created. A readme was placed in the latter to populate the folder.

## Why it does that

To make links work again.

## Further notes

No further notes.

## Linked Issue(s)

No issues was created.

### CODE REVIEW

**Review by ndr-brt — APPROVED** [review]
thanks

**Comment by ndr-brt** [comment]
please update the dependencies file, then we'll be ready to go

File: README.md [hunk 1]
```diff
@@ -11,5 +11,5 @@ captured as [decision records](docs/developer/decision-records/).
 
 ## Contributing
 
-See [how to contribute](https://github.com/eclipse-edc/Connector/blob/main/CONTRIBUTING.md) for
+See [how to contribute](https://github.com/eclipse-edc/.github/blob/main/CONTRIBUTING.md) for
```
Comment (juliapampus) [inline README.md:14]: We should link to the original file:
```suggestion
See [how to contribute](https://github.com/eclipse-edc/docs/blob/main/CONTRIBUTING.md) for
```

**Review by juliapampus — CHANGES_REQUESTED** [review]

**Review by juliapampus — APPROVED** [review]

**Comment by ndr-brt** [comment]
@rafaelmagalhaes-ctw stil not working, my suggestion would be to copy and paste the output of the failed job: https://github.com/eclipse-edc/Technology-Aws/actions/runs/6980195692/job/18995724372?pr=146

**Pull request merged**